vuepress/docs/next/docs/consume/identity-management.md

```diff
@@ -4,17 +4,16 @@ Entando Identity Management is based on open source Keycloak. Entando Applicatio
 
 ## Logging into your Keycloak Instance
 
-Keycloak is protected by a Secret deployed to your Entando Kubernetes instance. You can query Kubernetes for the Secret's default admin credentials, substituting your environment's namespace and Secret name:
+Keycloak is protected by a Secret deployed to your Entando Kubernetes instance. You can query Kubernetes for the Secret's default admin credentials via the following, modifying this command to use your environment's namespace and Secret name:
 
 ```
 ent kubectl get secret default-sso-in-namespace-admin-secret -n entando -o go-template="{{println}}Username: {{.data.username | base64decode}}{{println}}Password: {{.data.password | base64decode}}{{println}}{{println}}"
 ```
 
- To find the Secret name, run
+To find the Secret name, shell into your VM, then run the command below and search for the Secret that ends in `namespace-admin-secret`.
 ```
-kubectl get secrets -n YOUR-NAMESPACE
+sudo kubectl get secrets -n YOUR-NAMESPACE
```

Comment:
There are 2 kubectl commands on this page. Let's leave them both as kubectl (without ent) and make sure they have the -n setting so they match. The prod page right now doesn't match since one is ent kubectl -n entando and the other is kubectl -n YOUR-NAMESPACE, on the same page.